app/services/idv/steps/in_person/address_step.rb

```diff
@@ -21,6 +23,8 @@ def call
             next if attr == :same_address_as_id && capture_secondary_id_enabled?
             flow_session[:pii_from_user][attr] = flow_params[attr]
           end
+
+          maybe_redirect_to_verify_info if updating_address
```

Comment:
Nit: What do you think about refactoring `updating_address` to `updating_address?`?

Reply:
let's do that in a separate PR (and include `updating_state_id` ... @night-jellyfish already refactored `updating_ssn` in #8304)

Reply:
ohh, cool, I didn't notice, thanks @night-jellyfish!